### PR TITLE
App load tweaks, i18n and localStorage

### PR DESCRIPTION
**Pull request by t3chguy**
+ gen-i18n missed out `.ts` files
+ Analytics gets loaded by the languageHandler which is used in places like rageshakesetup so is needed to be "localstorage-safe" for when it is used to show a Modernizr failure.

### CODE REVIEW

File: src/Analytics.js [hunk 1]
```diff
@@ -123,8 +123,8 @@ const LAST_VISIT_TS_KEY = "mx_Riot_Analytics_lvts";
 
 function getUid() {
     try {
-        let data = localStorage.getItem(UID_KEY);
-        if (!data) {
+        let data = localStorage && localStorage.getItem(UID_KEY);
```
Comment (turt2live) [inline src/Analytics.js:126]: evidence from the test environment shows that this doesn't always work and can throw errors. Generally best to `typeof(localStorage) === 'object' && localStorage.getItem(whatever)` instead.

Comment (t3chguy) [reply src/Analytics.js:126]: Fair enough, this worked in my tests and matches the other null-guards in the app at least.

Comment (turt2live) [reply src/Analytics.js:126]: It's possibly only a NodeJS environment and browsers don't fail as annoyingly. https://github.com/matrix-org/matrix-js-sdk/pull/1300 was the recent failure in mind.

Comment (t3chguy) [reply src/Analytics.js:126]: going to merge as we use the same null-guard in most places. Will keep an eye out for it failing.

**Pull request merged**